packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -160,10 +166,17 @@ function initDataviz(channel) {
     .d3Force('x', d3.forceX())
     .d3Force('y', d3.forceY())
     .d3Force('collide', d3.forceCollide(graph.nodeRelSize()))
-    .linkWidth(0.2)
+    .linkWidth(
+      () => parseFloat(model.style.lineWidth) || styleFallback.lineWidth
```

Comment:
shouldn't the parsing happen when we assign the value to `model.style.lineWidth`?

Reply:
Good catch - I'll fix that, thanks!